app/src/main/java/org/mozilla/fenix/tabtray/TabTrayView.kt

```diff
@@ -95,6 +98,10 @@ class TabTrayView(
 
         view.tab_layout.addOnTabSelectedListener(this)
 
+        // We need to find a fix for this, this will require figuring out why the extended FAB won't
+        // respect any new elevation values
+        view.tab_wrapper.elevation = TEMP_ELEVATION
```

Comment:
What's the issue here?  Is the FAB displaying behind the tray or something?

Reply:
Yes